netket/graph/lattice.py

```diff
@@ -261,6 +177,17 @@ def __init__(
                 to :math:`k`-nearest neighbor sites (measured by their Euclidean distance)
                 are included in the graph. The edges can be distiguished by their color,
                 which is set to :math:`k - 1` (so nearest-neighbor edges have color 0).
+                By default, nearest neighbours (:code:`max_neighbor_order=1`) are autogenerated
+                unless :code:`custom_edges` is passed.
+            custom_edges: Lists all edges starting in one unit cell, which are repeated
```

Comment:
```suggestion
            custom_edges: (Optional) Lists all edges starting in one unit cell, which are repeated
```

I think this is useful to specify. It-s not like `max_neighbor_order` which has a default value. This argument is completely ignored if not specified. (Might be good to do the same for other arguments around here).

Also, i'd like if you could write down an example case of when you are supposed to use this. Like, to write down lattices that have XY and YZ blablabla